solana/lambda/scripts/multipleTransferTransaction.js

```diff
@@ -0,0 +1,127 @@
+import env from "dotenv";
+import { getLambdaSponsor } from "../../../lambda/lambdaUtils.mjs";
+import {
+  waitUntilTransactionMined,
+  checkAPIKey,
+  getAPIHost,
+} from "../../../helpers/utils.mjs";
+import { getDeveloperSolAddress } from "../../../helpers/utils.mjs";
+
+import Web3 from "@solana/web3.js";
+import nacl from "tweetnacl";
+async function main() {
+  env.config();
+
+  // Checks if the API_KEY is set in the .env file.
+  checkAPIKey();
+
+  /* *************************************************************** Invoke System Program *************************************************************** */
+  console.log(
+    "***************************************************** Invoke System Program *****************************************************\n"
+  );
+
+  console.log("Invoke solana transaction with multiple signatures...\n");
+  const developerSolAddress = await getDeveloperSolAddress();
+  console.log("Developer Solana Address: " + developerSolAddress);
+
+  const request = await transferSerializedTransaction(developerSolAddress);
```

Comment:
there's already invokeSolanaLambda in utils. Use that function instead.

Reply:
done